docs/topics/templates.txt

```diff
@@ -510,6 +530,116 @@ fictional ``foobar`` template library::
 
 See `DEP 182`_ for more information.
 
+Debug integration for custom engines
+------------------------------------
+
+.. versionadded:: 1.9
+
+    Debug page integration for non-Django template engines was added in Django
```

Comment:
since this renders as: "New in Django 1.9: ..." the "in Django 1.9" bit is redundant.